backend/entityservice/logger_setup.py

```diff
@@ -0,0 +1,92 @@
+import logging.config
+import os
+from pathlib import Path
+import structlog
+import yaml
+
+from entityservice.errors import InvalidConfiguration
+
+
+def setup_logging(
+    default_path='default_logging.yaml',
+    env_key='LOG_CFG'
+):
+    """
+    Setup logging configuration
+    """
+    path = os.getenv(env_key, Path(__file__).parent / default_path)
+    try:
+        with open(path, 'rt') as f:
+            config = yaml.safe_load(f)
+        logging.config.dictConfig(config)
+        msg = "Loaded logging config from file"
```

Comment:
It might make more sense to move this string literal to [line 32](https://github.com/n1analytics/entity-service/pull/349/files#diff-a6e1643a794254ae9581fb40cffdf0dcR32) where it is actually used.